specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,213 @@
+
+| Name        | Description                                                                                                 | Glyph                                |
+| ----------- | ----------------------------------------------------------------------------------------------------------- | ------------------------------------ |
+| None        | No glyph is displayed. The badge will appear blank or not be shown at all.                                  | (No Badge Shown)                     |
+| Activity    | A glyph indicating some form of activity is taking place within the app.                                    | ![Screenshot](badge-activity.png)    |
+| Alert       | A glyph that suggests an alert or an important notification that may require immediate attention.           | ![Screenshot](badge-alert.png)       |
+| Alarm       | A glyph that represents an alarm, possibly indicating a set reminder or a timed event.                      | ![Screenshot](badge-alarm.png)       |
+| Attention   | A glyph that signifies the need for attention, often used for notifications or new information.             | ![Screenshot](badge-attention.png)   |
+| Available   | A glyph that indicates the user or a service is available, often used in communication apps.                | ![Screenshot](badge-available.png)   |
+| Away        | A glyph that shows the user is away or inactive, commonly used in status indicators for communication apps. | ![Screenshot](badge-away.png)        |
+| Busy        | A glyph that represents the user being busy or engaged in an activity, preventing interruptions.            | ![Screenshot](badge-busy.png)        |
+| NewMessage  | A glyph that indicates the arrival of a new message, often used in messaging and email applications.        | ![Screenshot](badge-newmessage.png)  |
+| Paused      | A glyph that signifies a pause in activity or content, such as media playback being paused.                 | ![Screenshot](badge-paused.png)      |
+| Playing     | A glyph that indicates media or content is currently playing.                                               | ![Screenshot](badge-playing.png)     |
+| Unavailable | A glyph that shows the user or a service is not currently available or offline.                             | ![Screenshot](badge-unavailable.png) |
+| Error       | A glyph that denotes an error has occurred, which may require user action to resolve.                       | ![Screenshot](badge-error.png)       |
+
+These glyphs are system-provided and standardized, ensuring a consistent look and feel across
+different applications that use them. By using these glyphs, developers can convey specific states
+or notifications to users in a visually intuitive manner.
+
+> Note: Only system-provided badge images can be used.
+
+# API Details
+
+> Note: all of this new WinAppSDK API is to support Badge Notifications in WindowsAppSdk
+
+```c++ (but really MIDL3)
+namespace Microsoft.Windows.BadgeNotifications
```

Comment:
Why create a new namespace for .BadgeNotifications instead of putting these enum and classes into Microsoft.Windows.AppNotifications?  Is there a use-case for these outside of apps' badges?

(Adding to the existing namespace looks analogous to [BadgeUpdater ](https://learn.microsoft.com/en-us/uwp/api/windows.ui.notifications.badgeupdater?view=winrt-26100) being in Windows.UI.Notifications Namespace which is described as "Contains classes that encapsulate tile, toast, and badge notifications.".  Is WinAppSDK intentionally more granular?)